src/WpfMath/Rendering/IElementRenderer.cs

```diff
@@ -9,6 +9,16 @@ namespace WpfMath.Rendering
     /// <summary>Renderer interface for WPF-Math elements.</summary>
     public interface IElementRenderer
     {
+        /// <summary>
+        /// Gets the foreground of this renderer.
+        /// </summary>
+        Brush Foreground { get; }
+        
+        /// <summary>
+        /// Gets the background of this renderer.
+        /// </summary>
+        Brush Background { get; }
```

Comment:
As I said in another comment: let's remove these properties from the base interface. Implementations are free to keep them if required, though.